Rakefile

```diff
@@ -0,0 +1,133 @@
+require 'aws-sdk'
+
+CONFIGURATION   = 'Release'
+BUILD_DIR       = 'build'
+PROJECT_NAME    = 'TalkableSDK'
+UNIVERSAL_SDK   = "#{BUILD_DIR}/#{CONFIGURATION}-universal/#{PROJECT_NAME}.framework/#{PROJECT_NAME}"
+TMP_DIR         = 'tmp'
+DEMO_DIR        = 'example/TalkableSDKDemo'
+SDK_ARCHIVE     = "#{BUILD_DIR}/talkable_ios_sdk.zip"
+
+AWS_KEY         = 'AKIATIIUNDYPQ44V4GFD'
+AWS_SECRET      = '8VWPcR9XPPHbxX8kxii5FrgXjyGaOfNlc9x2/qwJ'
```

Comment:
@zhuravel @bogdan @dgtv please, make sure you have it changed ASAP

Reply:
@vitalikdanchenko Thank you. I deactivated this key.